snapcraft/commands/remote.py

```diff
@@ -257,17 +257,19 @@ def _run_new_remote_build(self) -> None:
                 "Do you wish to recover this build?", default=True
             ):
                 emit.progress("Building")
-                remote_builder.monitor_build()
-                emit.progress("Cleaning")
-                remote_builder.clean_build()
-                emit.progress("Build completed", permanent=True)
+                try:
+                    remote_builder.monitor_build()
+                finally:
```

Comment:
is this safe for the resume build feature?

Reply:
It has two scenarios:
1. Snapcraft got killed / reboot / network lost.
2. User Ctrl+C.

It should safe for the first one. But for the current code, if user Ctrl+C in the middle, it will run `cleanup()` and not resume-able. If we also want to resume it after Ctrl+C, it needs to remove the `finally`.